src/containers/App/Content.tsx

```diff
@@ -168,8 +168,7 @@ function DataWrapper({children}: {children: React.ReactNode}) {
     return (
         <GetUser>
             <GetNodesList />
-            <GetCapabilities />
-            {children}
+            <GetCapabilities>{children}</GetCapabilities>
```

Comment:
Wrapped everything with `GetCapabilities`, so nothing capability-base will be fetched before `/capabilities` loaded. In normal conditions this handler should work fast enough for this.